evento/app/controllers/events_controller.rb

```diff
@@ -20,16 +20,13 @@ def attendees
 
   # POST /events/:id/attendees
   def add_attendee
-    user = User.find(params[:user_id])
-    
-    if user != @current_user
-      render json: { error: 'Not Authorized' }, status: 401
-    elsif @event.attendees.include?(user)
-      render json: { message: 'You are already attendee of the event' }, status: 304
+    if @event.attendees.include?(@current_user)
+      render json: { message: 'You are already attendee of the event' }, status: 200
```

Comment:
Previous `304` code shouldn't be used here and for some reason it did not accept any messages. I find 200 more suitable.